### ISSUE TITLE
2.4.11 Przyjmowanie fokusu

### ISSUE THREAD

**Issue by zwiastunsw** [irdpl/wcag#18]
## Oryginał
````html
    <h4>Focus Appearance</h4>
    <p class="conformance-level">AA</p>
    <p class="change">New</p>
    <p>When the keyboard <a>focus indicator</a> is visible, one or both of the following are true:</p>
    <ol>
        <li>The entire <a>focus indicator</a> meets all the following:
            <ul>
                <li><a>encloses</a> the <a>user interface component</a>  or sub-component that is focused, and</li>
                <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states, and</li>
                <li>has a contrast ratio of at least 3:1 against adjacent non-focus-indicator colors.</li>
            </ul>
        </li>
        <li>An area of the focus indicator meets all the following:
            <ul>
                <li>is at least as large as the area of a 1 <a>CSS pixel</a> thick <a>perimeter</a> of the unfocused component 
                or sub-component, or is at least as large as a 4 CSS pixel thick line along the shortest side 
                of the <a>minimum bounding box</a> of the unfocused component or sub-component, and</li>
                <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states, and</li>
                <li>has a contrast ratio of at least 3:1 against adjacent non-focus-indicator colors, or is no thinner than 2 CSS pixels.</li>
            </ul>
        </li>
    </ol>
    <p>Exceptions:</p>
    <ul>
        <li>The focus indicator is determined by the user agent and cannot be adjusted by the author, or</li>
        <li>The focus indicator and the indicator's background color are not modified by the author.</li>
    </ul>
    <p class="note">What is perceived as the user interface component or sub-component (to determine enclosure or size) 
    depends on its visual presentation. The visual presentation includes the component's visible content, border, 
    and component-specific background. It does not include shadow and glow effects outside the component's content, 
    background, or border.</p>
    <p class="note">Examples of sub-components that may receive a focus indicator are menu items 
    in an opened drop-down menu, or focusable cells in a grid.</p>
    <p class="note">Contrast calculations can be based on colors defined within the technology (such as HTML, CSS and SVG). 
    Pixels modified by user agent resolution enhancements and anti-aliasing can be ignored.</p>
````
## Wstępna propozycja
````html
    <h4>Przyjmowanie fokusu</h4>
    <p class="conformance-level">AA</p>
    <p class="change">Nowe</p>
    
    <p>Jeśli <a>wskaźnik fokusu</a> jest widoczny, jedno lub oba z poniższych stwierdzeń są prawdziwe:</p>
    <ol>
        <li>Cały <a>wskaźnik fokusu</a> spełnia wszystkie poniższe warunki:
            <ul>
                <li><a>otacza</a> całość <a>komponentu interfejsu użytkownika</a>, komponentu podrzędnego,
                 na którym się skupia i</li>
                <li>ma <a>współczynnik kontrastu</a> 3:1 pomiędzy tymi samymi pikselami w sfokusowanym 
                i niesfokusowanym stanie i</li>
                <li>ma <a>współczynnik kontrastu</a> 3:1 względem przylegających kolorów niebędących częścią wskaźnika fokusu.</li>
            </ul>
        </li>
        <li>Obszar wskaźnika fokusu spełnia wszystkie poniższe warunki:
            <ul>
                <li>ma <a>obwód</a> o wielkości przynajmniej 1 <a>piksela CSS</a> grubości niesfokusowanego komponentu 
                 bądź komponentu podrzędnego lub jest wielkości przynajmniej 4 pikseli CSS grubości linii wzdłuż krótszego boku 
                 <a>minimalnej ramki zamykającej</a> niesfokusowanego komponentu bądź komponentu podrzędnego oraz</li>
                <li>ma <a>współczynnik kontrastu</a> 3:1 pomiędzy tymi samymi pikselami w sfokusowanym 
                i niesfokusowanym stanie oraz</li>
                <li>ma <a>współczynnik kontrastu</a> 3:1 względem przylegających kolorów niebędących częścią 
                 wskaźnika fokusu lub nie jest cieńszy niż 2 <a>piksele CSS</a></li>
            </ul>
        </li>
    </ol>
    <p>Wyjątki:</p>
    <ul>
        <li>Wskaźnik fokusu określony jest przez programy użytkownika i nie może być modyfikowany przez autora lub</li>
        <li>Wskaźnik fokusu i kolor tła wskaźnika nie są modyfikowane przez autora.</li>
    </ul>
    <p class="note">To, co postrzega się jako komponent interfejsu użytkownika bądź komponent podrzędny 
    (aby można było określić obramowanie lub rozmiar) zależy od prezentacji wizualnej. Prezentacja wizualna 
    zawiera widoczną treść komponentu, granice i specyficzny dla komponentu kolor tła. Nie zalicza się do niej 
    cieni i efektów poświaty poza treścią komponentu, jego tłem lub granicami</p>

    <p class="note">Przykładami komponentów podrzędnych, które mogą przyjmować wskaźnik fokusu, 
    są elementy listy rozwijanej menu lub pojedyncze komórki w siatce.</p>

    <p class="note">Współczynnik kontrastu można obliczyć w oparciu o kolory definiowane przez technologie 
    (takie jak HTML, CSS i SVG). Piksele modyfikowane przez programy użytkownika, wzmacniacze rozdzielczości 
    i anty-aliasing można zignorować.</p>
````
## Rzeczy do rozważenia
- **nazwa kryterium**: inna propozycja _Wygląd fokusu_
- **fokus**: a może jednak _skupienie_? Termin _fokus_ nie jest zdefiniowany w słowniku.
- **terminy słownikowe**. Rozważ spójność użytych terminów słownikowych z definicją w słowniku:  _wskaźnik fokusu_, _otacza_, _komponentu interfejsu użytkownika_, _obwód_, _piksela CSS_, _minimalnej ramki zamykającej_, _współczynnik kontrastu_
- **sfokusowany, niesfokusowany** - inne określenia: zogniskowany | niezogniskowany, skupiony | nieskupiony
---

**Comment by BartekStepien** [comment]
Nazwa kryterium - Wygląd fokusu.

**Comment by speplinski** [comment]
`Wygląd fokusa` czy `Wygląd fokusu`?

**Comment by zwiastunsw** [comment]
[Odmiana przez przypadki rzeczownika fokus](https://odmiana.net/odmiana-przez-przypadki-rzeczownika-fokus)
